src/components/ProjectCreation.js

```diff
@@ -1,68 +1,152 @@
+    }
+    
+    
+    let navigate = useNavigate(); 
+    
+    const projectSchema = Yup.object().shape({
+        name: Yup.string().max(30).required(),
+        leader: Yup.string().max(30).required(),
+        tags: Yup.string().max(30).required(),
+        description: Yup.string().max(30).required(),
+        skills: Yup.string().max(30).required(),
         
-<br></br>
-<Form>
-  <Form.Group className="Leader" controlId="formLeader">
-    <Form.Label>Project Leader Description</Form.Label>
-    <Form.Control type="leader" placeholder="Enter leader description" as ="textarea" rows="5"/>
-  </Form.Group>
+    });
 
-  <Form.Group className="Member" controlId="formMember">
-    <Form.Label>Current Members</Form.Label>
-    <Form.Control type="member" placeholder="Members" as ="textarea" rows ="5"/>
-  </Form.Group>
+    const projectClick = async (e) => {
+        e.preventDefault();
+        setError(false); 
+        setErrorMessage(''); 
+        const isValid = await projectSchema.isValid({
+            name: name,
+            leader: leader,
+            tags: tags,
+            description: description,
+            skills: skills,
+        });
 
-  <Form.Group className="Description" controlId="formDesc">
-    <Form.Label>Project Description</Form.Label>
-    <Form.Control type="project" placeholder="Enter project description" as ="textarea" rows = "15" />
-  </Form.Group>
+        if (isValid) {
+            axiosInstance
+            
+                .post('/projects', {
+                    name: name,
+                    leader: leader,
+                    tags: tags,
+                    description: description,
+                    skills: skills,
+                
+                }, {
+                    headers: {'authorization': 'Bearer ' + authorization},
+                })
+                .then((response) => {
+                    alert("Project has been created!");
+                    navigate('/projects/:id');
+                    console.log(response);
+                })
+                .catch((err) => {
+                    console.log(err);
+                    console.log(err.response);
+                    setErrorMessage(err.response.data.msg); 
+                    setError(true);
+                });
+        } else {
+            setError(true); 
+            setErrorMessage('One or more input is invalid');
+        }
+    };
 
-  <Form.Group className="Requirement" controlId="formReq">
-    <Form.Label>Project Requirement</Form.Label>
-    <Form.Control  type="requirement" placeholder="Enter project requirements" as ="textarea" rows ="10"/>
-  </Form.Group>
+    return (
+        <Form>
+            <Form.Group className = "Title" controlId = "formTitle">
+                <Form.Label>Project Title</Form.Label>
+                <Form.Control className = "name-input-css"
+                              type = "name"
+                              as = "textarea"
```

Comment:
Maybe simple text input as opposed to a textarea since a project title is typically short